osgp/platform/pom.xml

```diff
@@ -115,6 +115,10 @@
     <module>osgp-adapter-domain-tariffswitching</module>
     <module>osgp-adapter-domain-smartmetering</module>
 
+    <!-- kafka -->
```

Comment:
If you add these modules, you should probably have to add these two dependencies to components in shared  as well:
```
    <module>../shared/osgp-kafka-distributionautomation</module>
    <module>../shared/osgp-kafka-logger</module>
```

Reply:
These modules are already in the pom

Reply:
Are you sure, I do not see them in this pom?

Reply:
Oh, sorry, I was looking at the osgp-applications repo. They where added in that location. I just added them in this file.